lib/cambiatus/kyc/kyc.ex

```diff
@@ -27,6 +23,25 @@ defmodule Cambiatus.Kyc do
     end
   end
 
+  @doc """
+  Creates new KYC and address for a given account.
```

Comment:
This particular function clause doesn't use the address, only KYC.

```suggestion
  Creates a new KYC for a given account.
```